Multisig/UI/Transactions/TransactionsList/TransactionListView.swift

```diff
@@ -34,7 +34,9 @@ struct TransactionListView: Loadable {
             ForEach(model.transactionsList.sections) { section in
                 Section(header: SectionHeader(section.name)) {
                     ForEach(section.transactions) { transaction in
-                        NavigationLink(destination: TransactionDetailsView(transaction: transaction)) {
+                        // avoid reloading details on app open in order to
+                        // avoid crashing
+                        NavigationLink(destination: LoadableView(TransactionDetailsView(transaction: transaction), reloadsOnAppOpen: false)) {
```

Comment:
Did we align it with the product? It violates the requirement of background -> foreground data loading.

Reply:
currently it is  already disabled (because it was not using loadable view) - we discussed that yesterday in planning

Reply:
Ah, right. I thought you disabled it for the transactions list.
So disabling it for transaction details fixes the crash, right?

Reply:
yes